test/index.ts

```diff
@@ -0,0 +1,18 @@
+import { server } from '../src/server';
+import prisma from '../src/prisma';
+
+console.log(process.env.TEST_DATABASE_URL);
+
+before(async () => {
+  console.log('Server started successfully at http://localhost:4000');
+});
+
+after(async () => {
+  await prisma.$disconnect();
+  await server.stop();
+  console.log('Server stopped');
+});
+
+import './prisma-test';
```

Comment:
Não precisa de um caso de teste pra isso pq vc não está testando nada aqui. Vc deixar a conexão com o bnanco no `before` mesmo, pra que todos os testes tenham a possibilidade de usar o banco se assim for necessário.